### PR TITLE
Give buttons generic titles so they don’t mismatch casing

### PR DESCRIPTION
**Pull request by jordanmaguire**
## Update!

screen shot 2015-07-03 at 10 06 33 am

## Create!

screen shot 2015-07-03 at 10 06 39 am

### CODE REVIEW

**Comment by osiro** [comment]
:+1:

File: config/locales/simple_form.en.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 en:
+  helpers:
+    submit:
+      create: "Create"
```
Comment (pzi) [inline config/locales/simple_form.en.yml]: How do you feel about "Add" rather than "Create"? :)

Comment (jordanmaguire) [inline config/locales/simple_form.en.yml]: Not great - why do you prefer add?

Comment (pzi) [inline config/locales/simple_form.en.yml]: In my head it made more sense as you are 'adding' something to the existing set of 'xyz'. However, after doing some research I will put it down as a language thing.

http://ux.stackexchange.com/questions/53270/what-text-should-be-used-for-an-insertion-button-create-new-or-add

Comment (jordanmaguire) [inline config/locales/simple_form.en.yml]: :+1:

**Pull request merged**